infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -1218,11 +1218,9 @@
         "remark": {
             "changed-x-meters": "Muutos {{value}} m",
             "moved-x-meters": "Siirtynyt {{value}} m",
-            "changed-km-number": "Geometria ei siirtynyt. Muita tietoja muuttunut kilometriltä {{value}}",
-            "changed-km-numbers": "Geometria ei siirtynyt. Muita tietoja muuttunut kilometreiltä {{value}}",
+            "changed-km-number": "Muuttunut kilometriltä {{value}}",
+            "changed-km-numbers": "Muuttunut kilometreiltä {{value}}",
```

Comment:
Ei liity tämän tiketin juttuihin mitenkään, mutta tuli näitä käännösjuttuja aiemmin säätäessä vastaan, että ainakin i18next tukee monikkoja https://www.i18next.com/translation-function/plurals

Noiden käyttöönottoa voisi harkita (jollain omalla tiketillään,) joskin se tarkoittaisi myös sitä, että meidän täytyisi koodata vastaava tuki käsin meidän bäkkärin käännösmekanismiin.